lib/app/features/wallet/views/pages/manage_coins/manage_coins_page.dart

```diff
@@ -30,7 +31,11 @@ class ManageCoinsPage extends HookConsumerWidget {
 
     useOnInit(
       () {
-        ref.read(filteredCoinsNotifierProvider(searchText: searchText.value).notifier).filter(
+        ref
+            .read(
+              filteredCoinsNotifierProvider(searchText: searchText.value).notifier,
+            )
+            .filter(
               searchText: searchText.value,
             );
```

Comment:
auto format looks pretty odd, please fix it like this:
https://github.com/ice-blockchain/ion-app/pull/374#discussion_r1870051299

Reply:
Done